app/models/notification.rb

```diff
@@ -1,8 +1,7 @@
 class Notification < ApplicationRecord
```

Comment:
This class needs to remain for now so that we can run the migration to migrate our previous notifications over to the new noticed_notifications table. I have had to leave some validations in this for now for our database consistency checks. Once the migration is complete we can delete the notifications table and this model.